File: 2023-2024/sourcecode/mod2/lab9/RepositoryBasicsJava/app/src/main/java/com/example/repositorybasicsjava/repository/UserRepository.java
package com.example.repositorybasicsjava.repository;

import com.example.repositorybasicsjava.dummydata.DataProvider;
import com.example.repositorybasicsjava.model.User;

import java.util.List;

public class UserRepository {
    public List<User> getUsers(){
        return DataProvider.getUsers();
    }
}

File: 2023-2024/sourcecode/mod2/lab9/RepositoryBasicsCompose/app/src/main/java/com/example/repositorybasicscompose/repository/UserRepository.kt
package com.example.repositorybasicscompose.repository

import com.example.repositorybasicscompose.dummydata.DataProvider
import com.example.repositorybasicscompose.model.User
import kotlinx.coroutines.delay

class UserRepository {
    suspend fun getUsers(): List<User> {
        delay(700L)
        return DataProvider.users
    }
}